components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/DefaultURLResolverService.java

```diff
@@ -152,7 +169,7 @@ private void appendTenantAsPathParam(StringBuilder serverUrl) {
     private void appendTenantAsQueryParam(StringBuilder serverUrl) {
```

Comment:
Maybe we can rename this to indicate legacy mode operation as well so that it is consistent